.travis.yml

```diff
@@ -43,9 +37,15 @@ script:
   # Needed if no cache exists
   - mkdir -p $INTEGRATIONS_DIR
   - ls -al $INTEGRATIONS_DIR
-  - 'rake ci:run'
+  - 'rake ci:run[default]'
+  - 'rake ci:run[core_integration]'
+  - 'rake ci:run[checks_mock]'
+  - 'rake ci:run[system]'
   - ls -al $INTEGRATIONS_DIR
 
+after_success:
```

Comment:
should we do this after every run (failed or not)? This should not run with PR's so it's not something that will clobber our CI in the SDK repos (and CI activity in the `dd-agent` should come way down anyway, so it shouldn't be too bad).